src/pages/edge/lifecycle-for-edge-network/migration-reference.md

```diff
@@ -0,0 +1,62 @@
+
+<InlineAlert variant="info" slots="text"/>
+
+When using Lifecycle for Analytics, if the Analytics extension in Tags is configured with "backdate previous session info" and "offline enabled", then two events are dispatched on the start of a new session; one for the previous session info and one for the start of the new lifecycle session. If these configurations are not set, then only one event is dispatched for the start of the new lifecycle session.
+
+As you can see in the previous scenario, Lifecycle for Edge Network provides richer details on how a user interacts with an application. This level of granularity comes at a cost of more events being sent to the Edge Network than was previously sent when using Analytic directly.
+
+## Lifecycle data
+
+Lifecycle for Edge Network formats data to XDM, conforming to the [AEP Mobile Lifecycle Details](https://github.com/adobe/xdm/blob/master/docs/reference/adobe/experience/aep-mobile-lifecycle-details.schema.md) field group. The field group contains three subgroups: application details which are the Lifecycle data, environment details about the system software, and device details about the user's device. This Lifecycle data sent to the Edge Network is different in a couple key ways from the data sent from the mobile Analytics extension.
+
+1. Any metric or dimension which may be computed at reporting time from existing data is not included in Lifecycle for Edge Network events. This includes metrics such as number of launches, launches since upgrade, days since last use or launch, and dimensions such as events for daily or monthly user engagement, hour of day or day of week, and install date.
+2. The Edge Network extension handles more than just tracking events, as such Lifecycle dimensions are not sent on non lifecycle experience events as they are when using the mobile Analytics extension. These dimensions include operating system, device name and resolution, carrier name, run mode, and the application identifier. If necessary, application developers may add these dimensions to specific Edge Network requests either manually when calling [Edge.sendEvent()](../edge-network/api-reference.md#sendevent) or by configuring an attach data rule in Tags.
+
+Refer to the Lifecycle data reference documentation for a comparison between the [Lifecycle for Edge Network data](./metrics.md) and [Lifecycle for Analytics data](../../home/base/mobile-core/lifecycle/metrics.md).
+
+<InlineAlert variant="info" slots="text"/>
+
+The Adobe Experience Platform Edge Network automatically maps the Lifecycle XDM variables into Adobe Analytics with no additional configuration needed. When enabling the Analytic service in a datastream, the XDM Lifecycle variables for application, device, and environment are automatically mapped as defined in [XDM object variable mapping](https://experienceleague.adobe.com/en/docs/analytics/implementation/aep-edge/xdm-var-mapping).
```

Comment:
Should we also talk about how lifecycle data (like appId, etc..) is not sent with all the edge events and only on lifecycle events and not edge events. We have had customer issues for this type of questions as well so was thinking it would be better to add that section here.

Reply:
I see that message mentioned here: https://github.com/AdobeDocs/aep-mobile-sdkdocs/pull/736/files#diff-c4343bcb3f9de5b080f071a07189aecaa07c0939f85103f30eaeadc358349760R95. But it will be good to give some more information about it's implications.

Reply:
I do mention it in this page as well on line 56. It's a paragraph where I list the dimensions and state they are send only with the Lifecycle events. Do you think this information and format is clear enough?